frontend-shared/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
lint needs a second rc file here, I think because of the package.json existence, but I'm not 100% sure.